functions/src/models/requests/timeline.ts

```diff
@@ -81,6 +128,17 @@ export class TimelineItem implements ITimelineItem {
     this._requestSnapshot = value;
   }
 
+  @IsObject()
```

Comment:
@IsEnum() is better used here. We can pass in the enum to validate against.

Reply:
Done by 4788542